SConscript

```diff
@@ -13,7 +13,7 @@ if not env_etc.no_boost_python and hasattr(env_etc, "boost_adaptbx_include"):
     env = env_no_includes_boost_python_ext.Clone()
     env_etc.enable_more_warnings(env=env)
 
-    system_includes = [x for x in env_etc.conda_cpppath if x]
+    system_includes = [x for x in env_etc.conda_cpppath if x] if libtbx.env.build_options.use_conda else []
```

Comment:
From this line, I assume that `env_etc.conda_cpppath` is literally missing, instead of empty, outside of a conda build?

Reply:
Ya, the error message I get is something like 'empty has no attribute conda_cpppath'